llvm/lib/Target/AIE/AIEPostSelectOptimize.cpp

```diff
@@ -472,6 +539,12 @@ bool AIEPostSelectOptimize::runOnMachineFunction(MachineFunction &MF) {
     }
   }
 
+  // 4. Fix MMO for load instructions that don't use pointers
+  // registers (use vector instead, for example).
```

Comment:
nit: I think it adds MMOs for loads that don't already have them, irrespective of whether they are pointers or not.

Reply:
Changed the comment accordingly.